### PR TITLE
Add tests for vault and some small refactoring 

### PR DESCRIPTION
**Pull request by werne2j**
### Description
- Added some tests for the vault client, the config and the types.
- Did some refactor to set the token directly on the api client instead of passing it
- Updated the size of the logo 

### Checklist
Please make sure that your PR fulfills the following requirements: 
- [x] Reviewed the guidelines for contributing to this repository
- [x] The commit message follows the [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0/#summary).
- [x] Tests for the changes have been
- [ ] Docs have been added / updated

### Type of Change

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [x] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

### CODE REVIEW

File: pkg/vault/client_test.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+func TestVaultRead(t *testing.T) {
+	ln, client := CreateTestVault(t)
+	defer ln.Close()
+
+	vc := &Client{
+		VaultAPIClient: client,
+	}
+
+	t.Run("will get data from vault", func(t *testing.T) {
```
Comment (jkayani) [inline pkg/vault/client_test.go:16]: Didn't even know this was a thing

File: pkg/kube/secret.go [hunk 2]
```diff
@@ -16,8 +15,7 @@ type SecretTemplate struct {
 
 // NewSecretTemplate returns a *SecretTemplate given the template's data, and a VaultType
 func NewSecretTemplate(template map[string]interface{}, vault vault.VaultType) (*SecretTemplate, error) {
-	path := os.Getenv("VAULT_PATH_PREFIX")
-	data, err := vault.GetSecrets(path)
+	data, err := vault.GetSecrets("/secret")
```
Comment (jkayani) [inline pkg/kube/secret.go:18]: We will need this for `ConfigMap` too, I guess `/config`

**Review by jkayani — COMMENTED** [review]
Let's remove the Argo configMap and add the `GetSecrets("/config")` for the ConfigMap template and this is good

File: patch.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+data:
```
[the remark this answers is not in the document]
Comment (werne2j) [reply patch.yaml:1]: Yes, i want to be able to say, `apply patch file to cluster to setup the plugin" or whatever

**Comment by jkayani** [comment]
Actually, keep the Argo configMap. It can be useful to people trying to install this plugin in their Argo

**Review by jkayani — APPROVED** [review]
Good stuff!

**Pull request merged**